packages/dialtone-vue2/components/empty_state/empty_state.vue

```diff
@@ -0,0 +1,140 @@
+<template>
+  <div :class="emptyStateClasses">
+    <template v-if="showIllustration">
+      <dt-icon
+        v-if="showIcon"
+        :name="iconName"
+        size="800"
+      />
+
+      <dt-illustration
+        v-if="showIllustrationComponent"
+        :name="illustrationName"
+      />
+    </template>
+
+    <h1 v-if="headerText">
+      {{ headerText }}
+    </h1>
+
+    <p v-if="bodyText">
+      {{ bodyText }}
+    </p>
+
+    <slot name="body" />
+  </div>
```

Comment:
There seems to be a lot more spacing between image / body / header / buttons in the designs.

https://www.figma.com/design/2adf7JhZOncRyjYiy2joil/branch/XJ1R0cEOwJ8xTTAf4w8Zda/DT9-Component-Library?node-id=14939-84619&m=dev

Also there is padding on the container in the designs

Reply:
https://github.com/dialpad/dialtone/pull/315/commits/d3d5142a3417445c14ab857767c91ded54f9d3e5